pages/dkp/konvoy/2.2/choose-infrastructure/aws/advanced/prerequisites/index.md

```diff
@@ -19,7 +19,37 @@ Before you begin using Konvoy, you must have:
 
 <p class="message--note"><strong>NOTE: </strong>On macOS, Docker runs in a virtual machine. Configure this virtual machine with at least 8GB of memory.</p>
 
-If you use these instructions to create a cluster on AWS using the DKP default settings without any edits to configuration files or additional flags, your cluster is deployed on an [Ubuntu 20.04 operating system image][supported-systems] with 3 control plane nodes, and 4 worker nodes.
+## Control plane nodes
+
+You should have at least three control plane nodes.
+Each control plane node should have at least:
+
+- 4 cores
+- 16 GiB memory
+- Approximately 80 GiB of free space for the volume used for /var/lib/kubelet and /var/lib/containerd.
+- Disk usage must be below 85% on the root volume.
```

Comment:
In looking this over, I was thinking maybe that we could make this a little cleaner by rolling the default Amazon instance type into this section rather than listing it later.

I.e

```suggestion
- 4 cores
- 16 GiB memory
- Approximately 80 GiB of free space for the volume used for /var/lib/kubelet and /var/lib/containerd.
- Disk usage must be below 85% on the root volume.

DKP defaults to deploying an Amazon `m5.xlarge` instance with an 80GiB root volume for control plane nodes, which meets the above requirements.
```